index.html

```diff
@@ -1,19 +1,80 @@
 <!doctype html>
 
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <title>LXD UI</title>
-    <link rel="shortcut icon" href="/assets/img/favicon-32x32.png" type="image/x-icon">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script>const global = globalThis;</script>
-  </head>
-  <body>
+  <title>LXD UI</title>
+  <link rel="shortcut icon" href="/assets/img/favicon-32x32.png" type="image/x-icon">
+  <link rel="manifest" href="/manifest.json" id="manifest">
 
-    <div id="app"></div>
-    <script type="module" src="/src/index.tsx"></script>
+  <script>const global = globalThis;
+
+  </script>
+</head>
+
+<body>
+
+  <div id="app"></div>
+  <script type="module" src="/src/index.tsx"></script>
+
+</body>
 
-  </body>
 </html>
+
+<script>
+
+  var screenshot = window.location.origin + "/assets/img/";
```

Comment:
```suggestion
  var screenshot = window.location.origin + "/ui/assets/img/";
```
This needs to be under the ui route.